apps/trustlab/src/components/Card/Card.js

```diff
@@ -0,0 +1,167 @@
+import { Link } from "@commons-ui/next";
+import { LexicalRichText } from "@commons-ui/payload";
+import {
+  Box,
+  Typography,
+  Card as MuiCard,
+  CardActions,
+  CardContent,
+  CardHeader,
+  CardMedia,
+  Button,
+} from "@mui/material";
+
+import { neutral } from "@/trustlab/colors";
+
+function Card({
+  description,
+  link,
+  linkLabel,
+  media,
+  tag,
+  title,
+  DescriptionProps,
+  CardProps,
+  CardActionsProps,
+  CardContentProps,
+  CardHeaderProps,
+  CardMediaProps,
+  LinkProps,
+  TagProps,
+  TagTypographyProps,
+  TitleProps,
+}) {
+  return (
+    <MuiCard
+      {...CardProps}
+      sx={{
+        width: {
+          xs: "100%",
+          md: 336,
+        },
+        maxWidth: {
+          xs: "100%",
+          md: 336,
+        },
+        display: "flex",
+        flexDirection: "column",
+        height: "100%",
+        borderRadius: 0,
+        border: "1px solid",
+        borderColor: neutral[300],
+        ...CardProps?.sx,
```

Comment:
Use the array syntax so that all forms of `sx` works i.e. `function`, `Array`, or `object`.